### ISSUE TITLE
Missing Control Probes

### ISSUE THREAD

**Issue by nanabgh1** [zwdzwd/sesame#176]
Hi,
I’m having trouble using the DyeBiasCorr function and keep running into the following error: "Error in normControls(sdf, average = TRUE): No normalization control probes found!" I hope someone can help me with this. Thank you very much!

**Comment by zwdzwd** [comment]
Hi thanks for catching. I think it's a bug. I just fixed it in the latest upload (version 1.23.9). But I think you can use dyeBiasNL which is more recommended and used now. Let me know if you still see the issue after updating.

**Comment by nanabgh1** [comment]
Thanks for the quick answer! I still see the issue but dyeBiasNL works fine, thank you

**Comment by zwdzwd** [comment]
Great. Have you reached 1.23.9? dyeBiasCorr seems to work fine on my end.

**Comment by nanabgh1** [comment]
yes... sesame version 1.23.9, sesameData 1.22.0 and ExperimentHub 2.12.0

**Comment by zwdzwd** [comment]
Hmm, does the following work for you

```
sdf = sesameDataGet("EPICv2.8.SigDF")[[1]]
sdf1 = dyeBiasCorr(sdf)
```

**Comment by nanabgh1** [comment]
yes it does, Is the problem that in my sdf, the controls are listed as attr(* "controls") = 'data.frame'? If so, can you tell me how to resolve this? Thank you

**Comment by zwdzwd** [comment]
Ah, which platform EPIC? are you using? I forgot to ask.

**Comment by nanabgh1** [comment]
I want to analyze both, but currently I am working with HM450
